catalogue/webapp/components/Download/DownloadBeta.js

```diff
@@ -0,0 +1,245 @@
+// @flow
+import type { LicenseData } from '@weco/common/utils/get-license-info';
+import type { LicenseType } from '@weco/common/model/license';
+import { trackEvent } from '@weco/common/utils/ga';
+import { useState, useEffect } from 'react';
+import { convertImageUri } from '@weco/common/utils/convert-image-uri';
+import styled from 'styled-components';
+import { font, spacing, classNames } from '@weco/common/utils/classnames';
+import LicenseBeta from '@weco/common/views/components/License/LicenseBeta';
+import Icon from '@weco/common/views/components/Icon/Icon';
+
+const DownloadButton = styled.button`
+  text-align: center;
+  border: 0;
+  background: ${props => props.theme.colors.green};
+  color: ${props => props.theme.colors.white};
+  padding: ${props =>
+    `${props.theme.spacingUnit * 2}px ${props.theme.spacingUnit * 2}px ${props
+      .theme.spacingUnit * 2}px ${props.theme.spacingUnit * 3}px`};
+  display: inline-block;
+  margin: ${props =>
+    `0 ${props.theme.spacingUnit * 2}px ${props.theme.spacingUnit}px 0`};
+  cursor: pointer;
+  :focus {
+    outline: none;
+    box-shadow: 0 0 3px 3px rgba(0, 0, 0, 0.3);
+  }
+  .icon {
+    transition: transform 700ms;
+    transform: ${props =>
+      props.rotateIcon ? 'rotate(180deg)' : 'rotate(0deg)'};
+  }
+  .icon__shape {
+    fill: currentColor;
+  }
+`;
+const DownloadOptions = styled.div`
+  &.enhanced-styles {
+    border: ${props => `1px solid ${props.theme.colors.marble}`};
+    background: white;
+    box-shadow: 0 1px 6px 0 rgba(0, 0, 0, 0.3);
+    z-index: -1;
+    margin-top: ${props => `-${props.theme.spacingUnit}px`};
+    padding: ${props => `${props.theme.spacingUnit * 3}px`};
+    height: none;
+    opacity: 0;
+    transition: opacity 400ms;
+    position: absolute;
+  }
+  &.show {
+    z-index: 1;
+    opacity: 1;
+  }
+  li + li {
```

Comment:
We'd normally try to avoid this in BEM land, and just put the styles on the list items / anchor tag?
Does that still feel like a pattern we should be following? 

Or is there another pattern to be following? If we are going to follow 1, it would be good to document and spread the word so we're doing things 1 way.